middleware/middleware-1.3.19.md

```diff
@@ -0,0 +1,93 @@
+---
+slug: /release-notes/middleware/1.3.19
+title: Version 1.3.19
+---
+
+# fiskaltrust.Middleware 1.3.19 (Germany)
+_May 4, 2021_
+
+With this Middleware version, we added the possibility to switch the connected SCU for already started Queues. This change allows users of the Middleware to change the used TSE without having to create a new Queue. Additionally, we've included several important bug fixes in this update. 
+
+<div class="alert alert--warning" role="alert">Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.</div>
+
+## Feature: Switch connected TSE without creating a new Queue
+In the last weeks, we've already received lots of feedback and questions from partners who needed to switch TSEs due to numerous reasons. Until now, this was only possible indirectly by also creating a new Queue and retiring the old one. However, this leads to additional effort on all sides, and will be even more inconvenient when cloud TSEs that are currently in certification will need to be replaced by their certified follow-up version.
+
+We've therefore implemented a way to switch SCUs and therefore TSEs of existing Queues (SCUs are "bound" to a specific TSE and therefore cannot be re-used with other TSEs).
+
+In short, the switch process consists of these steps:
+1. The SCU switch is prepared in the portal, i.e. the target SCU is added to the Cashbox and marked as _switch target_.
+2. The Middleware is restarted to pull the latest changes after the config was rebuilt.
+3. The switch is initiated with an "initiate SCU switch" receipt. After this, no SCU is connected to the Queue anymore.
```

Comment:
do you think we should add the daily closing as it is needed to be able to proceed the initiate scu switch?

Reply:
Good one, thanks! Added that 👍